File: C/1037 - Intervalo.c
#include <stdio.h>
 
int main() {
 
  double X;
  scanf("%lf", &X);

  if (X < 0.00 || X > 100.00)
    printf("Fora de intervalo\n");
  else if (X <= 25.00)
    printf("Intervalo [0,25]\n");
  else if (X <= 50.00)
    printf("Intervalo (25,50]\n");
  else if (X <= 75.00)
    printf("Intervalo (50,75]\n");
  else
    printf("Intervalo (75,100]\n");
 
    return 0;
}

File: CSharp/1037 - Intervalo.cs
using System;

class URI {
  public static void Main (string[] args) {
    double V = Convert.ToDouble(Console.ReadLine());
		if (V >= 0 && V <= 25)
			Console.WriteLine("Intervalo [0,25]");
		else if (V >= 25.01 && V <= 50)
		  Console.WriteLine("Intervalo (25,50]");
		else if (V >= 50.01 && V <= 75)
		  Console.WriteLine("Intervalo (50,75]");
		else if (V >= 75.01 && V <= 100)
		  Console.WriteLine("Intervalo (75,100]");
		else
		  Console.WriteLine("Fora de intervalo");
  }
}